src/cs/Ssh/SshChannel.cs

```diff
@@ -115,6 +119,16 @@ public class SshChannel : IDisposable
 
 	public bool IsClosed => this.localClosed || this.remoteClosed;
 
+	/// <summary>
+	/// Gets the message that requested opening the channel.
+	/// </summary>
+	public ChannelOpenMessage OpenMessage { get; private set; }
```

Comment:
`set` is not needed for init-only properties.

Reply:
Updated.